metaflow/datatools/local.py

```diff
@@ -0,0 +1,165 @@
+import os
+
+from ..current import current
+from ..exception import MetaflowException
+from ..metaflow_config import DATATOOLS_LOCALROOT, DATATOOLS_SUFFIX
+from ..util import to_unicode
+
+
+class MetaflowLocalURLException(MetaflowException):
+    headline = "Invalid path"
+
+
+class MetaflowLocalNotFound(MetaflowException):
+    headline = "Local object not found"
+
+
+class LocalObject(object):
```

Comment:
What is the rationale for lifting this up from `includefile.py` into its own module? I don't think we will be providing an `s3()` like interface for the local filesystem to the end user anytime soon.

Reply:
Also, this fil can be thinned down considerably (e.g, L:125) since there is no expectation of parity in functionality with s3 datatool.